src/actions/emojis.js

```diff
@@ -94,17 +98,18 @@ export function getCustomEmojisByName(names) {
         const promises = [];
         names.forEach((name) => promises.push(getCustomEmojiByName(name)(dispatch, getState)));
 
-        return Promise.all(promises);
+        await Promise.all(promises);
+        return {data: true};
     };
 }
 
-export function getCustomEmojisInText(text) {
+export function getCustomEmojisInText(text: string): ActionFunc {
     return async (dispatch, getState) => {
         if (!text) {
             return {data: true};
         }
 
-        const state = getState();
+        const state: GlobalState = getState();
```

Comment:
Why `state: GlobalState` vs. declaring the parameter type on line 107?

Reply:
I think this is not needed, because getState is actually defined as part of the ActionFunc definition. So I can remove it :)

Reply:
Changed! Thanks :)